client/src/battle/battle_simulation.rs

```diff
@@ -1032,6 +1034,28 @@ impl BattleSimulation {
 
         sorted_entities.sort_by_key(|(_, key)| *key);
 
+        if let Ok((entity, character)) =
+            (self.entities).query_one_mut::<(&Entity, &Character)>(self.local_player_id.into())
+        {
+            let offset: Vec2 = entity.corrected_offset(self.perspective_flipped);
+            let tile_center =
+                (self.field).calc_tile_center((entity.x, entity.y), self.perspective_flipped);
+            let x_difference = 2.0;
+            let mut position = vec2(
```

Comment:
Might be able to shorten this by adding tile_center to this offset Vec2. `tile_center + vec2(...)`

Reply:
Looks like i should leave the review on a lower line, since the relevant stuff is below this snippet (same with the position += x_difference above)

Reply:
the tile center change to the Vec2 didn't work, unfortunately, but the x_difference change worked just fine.